suma-site.yml

```
@@ -138,7 +138,7 @@ content:
     # Select the branch you want to build
     # If you make changes on a branch and have the wrong branch selected you will not see your changes in your build
     branches:
-    - '.'
+    - 'new-migrationXY-toMaster-0rnela'
```

```suggestion
    - '.'
```
Good that you tested it :-)